bittide-instances/data/test_configs/default.yml

```diff
@@ -0,0 +1,10 @@
+# SPDX-FileCopyrightText: 2022-2023 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+.default:
+  dummy: "Tcl can not handle empty dictionaries, so we need a dummy entry here"
+
+tests:
+  # tests contains all the tests that must be run with their corresponding configuration
+  default_name:
+    dummy: "Tcl can not handle empty dictionaries, so we need a dummy entry here"
```

Comment:
```suggestion
.default: null  # Tcl does not support empty scalars, explicit null is needed

tests:
  # tests contains all the tests that must be run with their corresponding configuration
  default_name: null  # Tcl does not support empty scalars, explicit null is needed
```

Reply:
By the way, the Tcl `yaml` package version 0.4.1 has no problem with empty scalars, unfortunately, Vivado includes version 0.3.3.